src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -1184,6 +1184,13 @@ public override BoundNode VisitUnaryOperator(BoundUnaryOperator node)
             return UpdateExpression(builder, node.Update(node.OperatorKind, operand, node.ConstantValueOpt, node.MethodOpt, node.ResultKind, node.Type));
         }
 
+        public override BoundNode VisitReadOnlySpanFromArray(BoundReadOnlySpanFromArray node)
```

Comment:
>VisitReadOnlySpanFromArray [](start = 34, length = 26)

not related to this PR: I wonder  if there is a systematic way we can catch those situations (all the walkers/rewriters to consider when adding new bound nodes)... Maybe an analyzer could force us to write those methods, if only to call the base?